test/plugins/validate-semantic/2and3/schemas.js

```diff
@@ -88,6 +88,35 @@ describe(`validation plugin - semantic - 2and3 schemas`, () => {
         expect(firstError.path).toEqual(["paths", "/pets", "get", "parameters", "0"])
       })
     })
+    it("should return an error for a missing items value for an array schema in response in Swagger 2", () => {
+      const spec = {
+        swagger: "2.0",
+        "paths": {
+          "/pets": {
+            "get": {
+              "responses": {
+                "200": {
+                  schema: {
+                    name: "myParam",
+                    in: "query",
```

Comment:
Please remove `name` and `in` - responses do not use these keywords.

Reply:
Removed